apps/ledger-live-mobile/fastlane/Fastfile

```diff
@@ -243,6 +258,9 @@ platform :ios do
         gym(
           scheme: PROJECT_NAME,
           workspace: XCODE_WORKSPACE,
+          # This should come from env files - as we move each release type to its own app we will
+          # move them away from "Release" and to their own .env-file defined configuration
+          configuration: options[:nightly] ? ENV["APP_CONFIGURATION"] : "Release",
```

Comment:
same question here, i think the best practice is to rely on a single source of truth which is .env files

Reply:
👍 - see above